### DISCUSSION TITLE
Some question about generating orientation with textured fiber

### DISCUSSION THREAD

**Discussion by killuaLeo** [neperfepx/neper#836] [Q&A] [answered]
Hi All,

I am trying to generate the orientations with orientation fibers. I followed the way in the documentation but it's not working. The command I am using is shown below:

`neper -T -n 200 -domain "cube(0.01,0.01,0.01)" -morpho gg -crysym hexagonal -ori fiber(0,0,1,1,0,0)[:normal(thetam = 10)] -o textured -format ori`

The error message is: bash: syntax error near unexpected token `(' . I am not sure if it is because I misunderstand the documentation in NEPER. I will be very appreciated if anyone could give some examples on how to generate orientations with orientation fibers. Thanks a lot!

**Comment by rquey** [comment, marked as answer]
```
neper -T -n 200 -domain "cube(0.01,0.01,0.01)" -morpho gg -crysym hexagonal -ori "fiber(0,0,1,1,0,0):normal(thetam=10)" -o textured -format ori
```

`[` and `]` indicate an optional argument and aren't to be used in the actual command.
